packages/back-end/generated/spec.yaml

```diff
@@ -664,6 +609,245 @@ paths:
+            curl -X POST https://api.growthbook.io/api/v1/metrics \
+              -d '{ datasourceId: "ds_abc123", ... }'
+              -u secret_abc123DEF456:
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required:
+                - datasourceId
+                - name
+                - type
+              properties:
+                datasourceId:
+                  type: string
+                  description: 'ID for the [DataSource](#tag/DataSource_model)'
+                owner:
+                  type: string
+                  description: Name of the person who owns this metric
+                name:
+                  type: string
+                  description: Name of the metric
+                description:
+                  type: string
+                  description: Description of the metric
+                type:
+                  type: string
+                  description: 'Type of metric. See [Metrics documentation](/app/metrics)'
+                  enum:
+                    - binomial
+                    - count
+                    - duration
+                    - revenue
+                tags:
+                  type: array
+                  description: List of tags
+                  items:
+                    type: string
+                projects:
+                  type: array
+                  description: List of project IDs for projects that can access this metric
+                  items:
+                    type: string
+                archived:
+                  type: boolean
+                behavior:
+                  type: object
+                  required: []
+                  properties:
+                    goal:
+                      type: string
+                      enum:
+                        - increase
+                        - decrease
+                    cap:
+                      type: number
+                      description: This should be non-negative
+                    conversionWindowStart:
+                      type: number
+                      description: 'The start of a Conversion Window relative to the exposure date, in hours. This is equivalent to the [Conversion Delay](/app/metrics#conversion-delay)'
+                    conversionWindowEnd:
+                      type: number
+                      description: 'The end of a [Conversion Window](/app/metrics#conversion-window) relative to the exposure date, in hours. This is equivalent to the [Conversion Delay](/app/metrics#conversion-delay) + Conversion Window Hours settings in the UI. In other words, if you want a 48 hour window starting after 24 hours, you would set conversionWindowStart to 24 and conversionWindowEnd to 72 (24+48).'
+                    riskThresholdSuccess:
+                      type: number
+                    riskThresholdDanger:
+                      type: number
```

Comment:
Let's add 

`riskThresholdDanger`: "Threshold for Risk to be considered too high, as a proportion (e.g. put 0.0125 for 1.25%)"
`riskThresholdSuccess`: "Threshold for Risk to be considered low enough, as a proportion (e.g. put 0.0025 for 0.25%)"

Reply:
Addressed in [9af9ec6](https://github.com/growthbook/growthbook/pull/1116/commits/9af9ec620facaffde2bfdc68fdf60e02e7ee81ce)